getting-started/multiband-analysis.rst

```diff
@@ -30,27 +32,32 @@ Ensure that this Python session is running from the shell where you ran :command
 Loading forced photometry measurement catalogs with the Butler
 ==============================================================
 
-The :command:`forcedPhotCoadd.py` command-line task (:ref:`getting-started-tutorial-forced-coadds`) created ``deepCoadd_forced_src`` datasets for each coadd in the example data repository.
+The ``forced_objects`` pipeline (:ref:`getting-started-tutorial-forced-coadds`) created ``deepCoadd_forced_src`` datasets for each coadd in the example data repository.
 Being forced photometry catalogs, rows in each ``deepCoadd_forced_src`` table correspond row-for-row across all coadds in different filters for a sky map patch.
```

Comment:
Another sky map which might need changing depending on whether it is sky map or skymap.